libs/perun/dialogs/src/lib/delete-entity-dialog/delete-entity-dialog.component.ts

```diff
@@ -1,19 +1,19 @@
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { FormControl, Validators } from '@angular/forms';
 import { MatTableDataSource } from '@angular/material/table';
-import { Facility, Group, Resource, Vo } from '@perun-web-apps/perun/openapi';
+import { Facility, Group, Resource, User, Vo } from '@perun-web-apps/perun/openapi';
 
 @Component({
-  selector: 'app-delete-entity-dialog',
+  selector: 'perun-web-apps-delete-entity-dialog',
   templateUrl: './delete-entity-dialog.component.html',
   styleUrls: ['./delete-entity-dialog.component.scss'],
 })
-export class DeleteEntityDialogComponent {
+export class DeleteEntityDialogComponent implements OnInit {
   @Input()
   title: string;
   @Input()
-  entityNames: MatTableDataSource<Facility | Group | Vo | Resource> = new MatTableDataSource<
-    Facility | Group | Vo | Resource
+  entityNames: MatTableDataSource<Facility | Group | Vo | Resource | User> = new MatTableDataSource<
```

Comment:
Create an allias for the type of this data source. `Service` should be part of the union as well.